ui/src/components/admin/fundraise/AdminFundraiseContainer.js

```diff
@@ -120,8 +117,7 @@ export default function FundraiseContainer(props) {
   };
   const onSubmit = async (event) => {
     event.preventDefault();
-    if (!campaignName || campaignName.length === 0 || campaignNameCheck())
```

Comment:
what was this for actually lol not sure why it's removed

Reply:
1 campaign can have multiple categories, so it needs more than 1 object. so campaignName has to be duplicated. if u look at the backend, i actually just create another category called 'pledge'